css/media.css

```diff
@@ -0,0 +1,1039 @@
+@media screen and (max-width: 1100px) {
+  .container {
+    max-width: 970px;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654145)